gf-webapp/src/component/pageBorder/PageBorder.tsx

```diff
@@ -7,9 +7,11 @@ import {loadDictionary} from "src/service/Language/LoadDictionary";
 import {Language, languageList} from "src/model/Language";
 import {useLanguage} from "src/service/Language/useLanguage";
 import styles from "src/component/pageBorder/PageBorder.module.scss";
+import {Loader} from "gf-ui-lib/src/components/Loader/Loader";
 
 interface PageBorderProps {
   children: ReactNode
```

Comment:
no need to explicitly specify children in props

Reply:
I don't understand. Right now, I only import the Loader once in PageBorder. Otherwise, I have to import it into every component in which it is used. This is the solution I wrote before the review. You are contradicting each other.

Reply:
If I need props isLoading, can I write like this?
`interface PageBorderProps extends PropsWithChildren{
  isLoading?: boolean
}`
`export const PageBorder = (props: PageBorderProps): ReactElement => {
      <Loader isLoading={props.isLoading} />
}`